Pages/email-verification.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" intial-scale="1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" type="text/css" href="../scss/layout/register.css" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <link rel="shortcut icon" href="../assets/ECX.svg" type="image/x-icon" />
+    <link rel="stylesheet" href="../scss/main.css"/>
+    <link rel="shortcut icon" href="assets/ECX.svg" type="image/x-icon"/>
+    <title>Password Reset</title>
+  </head>
+  <body>
+    <main>
+      <nav>
+        <section class="navbar_first">
+          <div class="navbar_ecx">
+            <a href="../index.html"
+              ><img class="navbar_0ecx1" src="../assets/ECX.svg" alt="ECX_LOGO"
+            /></a>
+          </div>
+          <div class="navbar_sep">
+            <div class="navbar_reg1">
+              <a class="navbar_reg4" href="Register.html">REGISTER</a>
+            </div>
+            <div class="navbar_ham">
+              <span class="navbar_myham">&#9776;</span>
+            </div>
+          </div>
+        </section>
+  
+        <section class="navbar_second">
+          <div class="navbar_about"><a href="../index.html">Home</a></div>
+          <div class="navbar_about"><a href="About.html">About</a></div>
+  
+          <div class="navbar_ment">
+            <a href="../index.html#mentors">Mentors</a>
+          </div>
+          <div class="navbar_sch">
+            <a href="../index.html#schedule">Schedule</a>
+          </div>
+          <div class="navbar_sch"><a href="../index.html #contact">Contact</a></div>
+          <div class="navbar_reg2">
+            <a class="navbar_reg3" href="Register.html">REGISTER</a>
+          </div>
+          <div class="navbar_log">
+            <a class="navbar_log2" href="Login.html">LOGIN</a>
+          </div>
+        </section>
+      </nav>
+    <main class="register-main-container">
+      <div class="register-form-container login-container">
+        <form class="login-form">
+          <br>
+          <br>
+          <div class="password-reset-link" id="passwordResetLink">
+            <h2 class="sent-reset-link">a password reset link will be sent to your email address</h2>
```

Comment:
lol the 'email' should be dynamic. The email you collect from the previous page, and change the className please.